### DISCUSSION TITLE
Vercel error with useContext(ThemeContext)

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#700] [Help]
I keep on getting this error on vercel when compiling a TS file without any additional settings:

`Argument of type 'import("/vercel/path0/node_modules/@types/react-dom/node_modules/@types/react/index").Context<import("/vercel/path0/node_modules/@types/styled-components/index").DefaultTheme>' is not assignable to parameter of type 'React.Context<import("/vercel/path0/node_modules/@types/styled-components/index").DefaultTheme>'.  TS2345`

Is there a fix to this?

**Comment by officialrajdeepsingh** [comment]
Share code with error. so we can help you more accurately
